proxies/live/apiproxy/proxies/default.xml

```diff
@@ -1,5 +1,21 @@
 <ProxyEndpoint name="default">
-  <PreFlow/>
+  <PreFlow>
+    <Request>
+      <Step>
+          <Name>AssignMessage.Errors.Default</Name>
+      </Step>
+      <Step>
+          <Name>AssignMessage.Copy.RequestHeaders</Name>
+      </Step>
+    </Request>
+  </PreFlow>
+  <PostFlow>
+    <Response>
+      <Step>
+        <Name>AssignMessage.Revert.RequestHeaders</Name>
```

Comment:
Why aren't you using 
```
            <Step>
                <Name>AssignMessage.Mirror.CorrelationID</Name>
                <Condition>(original-request-details.header.X-Correlation-ID ~~ ".+")</Condition>
            </Step>
            <Step>
                <Name>AssignMessage.Mirror.RequestID</Name>
                <Condition>(original-request-details.header.X-Request-ID ~~ ".+")</Condition>
            </Step>
```

Reply:
I guess mainly, so it is saying what is happening - in this case is it reverting the headers back to the original X- state - so these can be picked up by the Splunk logging.